src/PostgREST/Error.hs

```diff
@@ -55,18 +56,18 @@ data ApiRequestError
   | InvalidBody ByteString
   | ParseRequestError Text Text
   | NoRelBetween Text Text
-  | AmbiguousRelBetween Text Text [Relation]
+  | AmbiguousRelBetween Text Text [Relationship]
   | InvalidFilters
   | UnacceptableSchema [Text]
-  | UnknownRelation                -- Unreachable?
+  | UnknownRelationship                -- Unreachable?
```

Comment:
`grep` tells me we're really not using that at all - let's remove it right away, while it shows up here in the diff? :)

Reply:
Done!